app/components/after_sign_off_component.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class AfterSignOffComponent < ViewComponent::Base
+  def initialize(planning_application:)
+    @planning_application = planning_application
+    return unless render?
+
+    @recommendation = planning_application.recommendation
+    @challenged = @recommendation.challenged
+    @accessor_name = @recommendation.assessor.name
+  end
+
+  def render?
```

Comment:
I think it's cleaner than having logic branch around the component as the logic to display or not is shared externally and internally. This way all the logic is internal and tested.